composites/AlgoliaSearch/SearchResults.js

```diff
@@ -0,0 +1,202 @@
+import React from "react";
+import PropTypes from "prop-types";
+import a11ySpeak from "a11y-speak";
+import { injectIntl, intlShape, defineMessages } from "react-intl";
+import styled from "styled-components";
+import { ZebrafiedListTable } from "../basic/Table/ListTable";
+import { Row } from "../basic/Table/Row";
+import colors from "../../style-guide/colors.json";
+
+const messages = defineMessages( {
+	noResultsText: {
+		id: "searchresults.noresultstext",
+		defaultMessage: "No results found.",
+	},
+	foundResultsText: {
+		id: "searchresults.foundresultstext",
+		defaultMessage: "Found {resultsCount} { resultsCount, plural, one {result} other {results} }",
```

Comment:
for better translations, this should use placeholders and replacements. Or, to simplify, a text like:
`Number of results found: %d` is considered acceptable even for languages that have multiple plural forms

Reply:
Shouldn't we use the tools that `react-intl` provides us?

Reply:
`Number of results found: %d` seems easier to me, and is consistent with My Yoast